chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/ValidDataReviewer.java

```diff
@@ -14,7 +15,9 @@
 @Component("validDataReviewer")
 public class ValidDataReviewer implements Reviewer {
 
-    @Autowired private CertifiedProductManager cpManager;
+    @Lazy
+    @Autowired
+    private CertifiedProductManager cpManager;
```

Comment:
@Lazy works for constructors too. Consider putting autowiring the class constructor if it's an option. If not, no biggie.